Add 11.3 release notes (#147)
mondoohq/docs: releases/2024-05-07-mondoo-11.3-is-out.md

---
slug: mondoo-11.3-is-out/
title: Mondoo 11.3 is out!
description: Announcing the 11.3 release of Mondoo, the security and compliance platform that prioritizes risks that matter most in your infrastructure.
author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
tags: [release, mondoo]
---

> ## 🥳 Mondoo 11.3 is out! This release includes new Kubernets policies, GitHub org scanning support, and more!

Reviewer: ```suggestion
## 🥳 Mondoo 11.3 is out! This release includes new Kubernetes policies, GitHub org scanning support, and more!
```

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### All-new CIS Kubernetes policies

Mondoo now includes the latest CIS Kubernetes benchmark policies for self-managed Kubernetes clusters, EKS, AKS, and GKE. These policies include the latest CIS recommendations as well as all-new queries for improved output so you can remediate issues more quickly.

> ### GitHub Organization scanning

Reviewer: ```suggestion
### GitHub organization scanning
```

> Set it and forget it with complete GitHub Organization scanning. Configure the integration one time and scan new repositories as they're created so you're never caught off guard.

Reviewer: ```suggestion
Set it and forget it with complete GitHub organization scanning. Configure the integration one time and scan new repositories as they're created so you're never caught off guard.
```

DEETS IMAGE

Need more control over what's scanned? Specify individual repositories to include in the scan, or repos you'd like skipped.

DEETS IMAGE

Scanning on the command line? Enjoy a 10x performance improvement and new scalability improvements to scan large organizations.

## 🧹 IMPROVEMENTS

### Better understand risks of affected assets

> CVE and advisory pages now include the count of invidual risk factors, so you can better understand the distribution of risk throughout your infrastructure.

Author: ```suggestion
CVE and advisory pages now include the count of individual risk factors, so you can better understand the distribution of risk throughout your infrastructure.
```

DEETS IMAGE

### Cancel running AWS integration scans

> Cancel all running AWS instance scans for your organization or account directly from the integration page with a new "Cancel Scans" option in the 3-dot menu.

Reviewer: ```suggestion
Cancel all running AWS instance scans for your organization or account directly from the integration page with a new "Cancel Scans" option on the ellipsis menu.
```

DEETS: SCREENSHOT HERE

### Resource updates

#### github.organization

- Add `hasOrganizationProjects` field
- Add `hasRepositoryProjects` field

## 🐛 BUG FIXES AND UPDATES

- Prevent empty asset names when scanning operating systems.

> - Don't fail when using the junit output formatter if a policy bundle is empty.

Reviewer: ```suggestion
- Don't fail when using the JUnit output formatter if a policy bundle is empty.
```

- Don't require delete run commands permission to scan VMs in Azure.
- Fix failures in the `aws.elb.loadbalancer` resource when used with `--discover resources`.
- Improve asset overview data for various AWS assets when scanning with `--discover resources`.
- Improve performance of asset platform detection.
- Improve the space dashboard experience when scanning VMware and Azure assets.

> - Better handle empty author and commiter data in GitHub repo scans.

Reviewer: ```suggestion
- Better handle empty author and committer data in GitHub repo scans.
```

- Fix a failure loading some asset scans.
- Fix breadcrumbs on CVE and advisory pages.
- Improve rendering of the affected assets page risk factors on narrow displays.
- Improve rendering of inventory list platform badges on narrow displays.

> - Show more check description text on individual check pages before truncating it with a Show More link.

Reviewer: ```suggestion
- On individual check pages, show more of the check description text before truncating it with a Show More link.
```

> - Use a more clear icon for generating compliance reports from framework pages.

Reviewer: ```suggestion
- Use a clearer icon for generating compliance reports from framework pages.
```
